tiatoolbox/models/architecture/mapde.py

```diff
@@ -0,0 +1,294 @@
+"""Defines MapDe architecture.
+
+Raza, Shan E. Ahmed, et al. "Deconvolving convolutional neural network
+for cell detection." 2019 IEEE 16th International Symposium on Biomedical
+Imaging (ISBI 2019). IEEE, 2019.
+
+"""
+
+import numpy as np
+import torch
+import torch.nn.functional as F  # noqa: N812
+from skimage.feature import peak_local_max
+
+from tiatoolbox.models.architecture.micronet import MicroNet
+from tiatoolbox.utils.misc import select_device
+
+
+class MapDe(MicroNet):
+    """Initialize MapDe [1].
+
+    The following models have been included in tiatoolbox:
+
+    1. `mapde-crchisto`:
+        This model is trained on `CRCHisto dataset
+        <https://warwick.ac.uk/fac/cross_fac/tia/data/crchistolabelednucleihe/>`_
+    2. `mapde-conic`:
+        This model is trained on `CoNIC dataset
+        <https://conic-challenge.grand-challenge.org/evaluation/challenge/leaderboard//>`_
+        Centroids of ground truth masks were used to train this model.
+        The results are reported on the whole test data set including preliminary
+        and final set.
+
+    The tiatoolbox model should produce the following results on the following datasets
+    using 8 pixels as radius for true detection:
+
+    .. list-table:: MapDe performance
+       :widths: 15 15 15 15 15
+       :header-rows: 1
+
+       * - Model name
+         - Data set
+         - Precision
+         - Recall
+         - F1Score
+       * - mapde-crchisto
+         - CRCHisto
+         - 0.81
+         - 0.82
+         - 0.81
+       * - mapde-conic
+         - CoNiC
```

Comment:
```suggestion
         - CoNIC
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/443/commits/4dcf9d765d82e7918ff026fd60b48af65cbb89da